nginx/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM jonasal/nginx-certbot:5.4
+
+ENV NGINX_DIR=/etc/nginx
+
+RUN apt-get update &&  apt-get install --no-install-recommends --no-install-suggests -y \
+ wget
+
+# https://github.com/mitchellkrogza/nginx-ultimate-bad-bot-blocker/blob/master/MANUAL-CONFIGURATION.md
```

Comment:
Is there any reason for doing manual installation rather than using [auto installation scripts](https://github.com/mitchellkrogza/nginx-ultimate-bad-bot-blocker/blob/master/AUTO-CONFIGURATION.md) here?

Reply:
I've checked the auto installation scripts. `install-ngxblocker` works for installing the necessary files, and can replace lines 8-19 if needed. However I can't get `setup-ngxblocker` to work properly, so the include directives still need to be added manually ([as shown here](https://github.com/depositar/depositar-docker/pull/1/files/f8f2fdff818e39bdc14e36c33d7987f584ae8815#diff-c2e61af6cd6e0bf008dadceb0953247dc3203f8a6e298dc89d16e273ef6a9c24R60-R64)).

Reply:
> However I can't get `setup-ngxblocker` to work properly, so the include directives still need to be added manually ([as shown here](https://github.com/depositar/depositar-docker/pull/1/files/f8f2fdff818e39bdc14e36c33d7987f584ae8815#diff-c2e61af6cd6e0bf008dadceb0953247dc3203f8a6e298dc89d16e273ef6a9c24R60-R64)).

Could you try to specify the configuration directory via `./setup-ngxblocker -c /etc/nginx/user_conf.d`?

Reply:
Please rename the `default.conf` in the `nginx/user_conf.d` to `default.vhost`, then run the command `./setup-ngxblocker -v /etc/nginx/conf.d`.

PS. we rename the `default.conf` because there are other `.conf` in the `/etc/nginx/conf.d`.

Reply:
This is because the `default.conf` only appears in the `/etc/nginx/conf.d` (the real path used by NGINX) when the container is running (via a [symbolic link](https://github.com/JonasAlfredsson/docker-nginx-certbot/blob/master/src/scripts/util.sh#L254)). Therefore, the `setup-ngxblocker` doesn't work during image creation.

So let's keep the lines added by the `setup-ngxblocker` script in the `default.conf`, just like what you've written so far.